### PR TITLE
handle project comments turned off

### PR DESCRIPTION
**Pull request by picklesrus**
Adds the CommentStatus box to project pages when comments are turned off. 
Here's a screenshot:
image

### CODE REVIEW

File: src/views/preview/presentation.jsx [hunk 1]
```diff
@@ -574,72 +574,91 @@ const PreviewPresentation = ({
...
                                         ) : null}
                                     </FlexRow>
-
-                                    {/* Do not show the top-level comment form in single comment mode */}
-                                    {!singleCommentId && (
-                                        <FlexRow className="comments-root-reply">
-                                            {projectInfo.comments_allowed ? (
-                                                isLoggedIn ? (
-                                                    isShared && <ComposeComment
+                                    {isProjectCommentsGloballyEnabled ? (
```
Comment (picklesrus) [inline src/views/preview/presentation.jsx:577]: This part of the diff is much easier to read if you turn off whitespace diffs.

File: src/views/preview/presentation.jsx [hunk 2]
```diff
@@ -36,7 +37,6 @@ const projectShape = require('./projectshape.jsx').projectShape;
 require('./preview.scss');
 
 const frameless = require('../../lib/frameless');
-
```
Comment (benjiwheeler) [inline src/views/preview/presentation.jsx:39]: was removing this line intentional?

File: src/views/preview/presentation.jsx (hunk 1 above)
Comment (benjiwheeler) [reply src/views/preview/presentation.jsx:577]: Thanks!!

File: src/views/preview/project-view.jsx [hunk 3]
```diff
@@ -109,6 +110,7 @@ class Preview extends React.Component {
             socialOpen: false,
             favoriteCount: 0,
             isProjectLoaded: false,
+            isProjectCommentsGloballyEnabled: false,
```
Comment (benjiwheeler) [inline src/views/preview/project-view.jsx:113]: i think you want to remove this from the state object, right?

Comment (picklesrus) [reply src/views/preview/project-view.jsx]: Yes! Thank you.

File: src/views/preview/presentation.jsx (hunk 2 above)
Comment (picklesrus) [reply src/views/preview/presentation.jsx]: Nope. Put it back!

> I know we're not super consistent, but I think in project-view.jsx you should add `isProjectCommentsGloballyEnabled: false` to `DefaultProps`
Done!

**Comment by benjiwheeler** [comment]
LGTM

**Comment by kenzievt** [comment]
hello

hello my name is Kenzie did you mean to message me
________________________________

**Comment by kenzievt** [comment]
can you scratch team please help me

**Review by kenzievt — COMMENTED** [review]
good

**Comment by kenzievt** [comment]
okay can you help me with my email address please

**Review by kenzievt — COMMENTED** [review]
goood

**Pull request merged**